src/components/LeaderBoard/LeaderBoardChart.tsx

```diff
@@ -40,10 +41,12 @@ const LeaderBoardChart: React.FC = () => {
         data: data.map((col) =>
           col.totalStep ? col.totalStep : col.avg_steps
         ),
+        backgroundColor: data.map((col) => col.highlight ? 'rgba(45, 211, 111, 0.5)' : 'rgba(226, 127, 38, 0.5)'),
```

Comment:
I really really like this solution. Where do you pull these colors from? Are they from the walktober color pallette or just colors that you came up with?